### PR TITLE
Refactor implicit copy modules to include more configurable helm variables

### PR DESCRIPTION
**Pull request by froesef**
…a (to have the same defaults as the mover)

Signed-off-by: Florian Froese <ffr@zurich.ibm.com>

### CODE REVIEW

File: modules/m4d-implicit-copy-stream/templates/streamtransfer.yaml [hunk 1]
```diff
@@ -61,3 +61,6 @@ spec:
   imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
   {{ end }}
   noFinalizer: {{ .Values.noFinalizer }}
+  readDataType: "logdata"
```
Comment (shlomitk1) [inline modules/m4d-implicit-copy-stream/templates/streamtransfer.yaml:64]: LogData

File: modules/m4d-implicit-copy-stream/templates/streamtransfer.yaml [hunk 2]
```diff
@@ -61,3 +61,6 @@ spec:
...
+  writeDataType: "logdata"
+  writeOperation: "append"
```
Comment (shlomitk1) [inline modules/m4d-implicit-copy-stream/templates/streamtransfer.yaml:66]: Append

File: modules/m4d-implicit-copy-stream/values.yaml [hunk 3]
```diff
@@ -35,3 +35,7 @@ copy:
 #        object_key: ""
 #    format: ""
 #    vault: {}
+
+readDataType: "ChangeData"
```
Comment (shlomitk1) [inline modules/m4d-implicit-copy-stream/values.yaml:39]: Didn't you want to change to LogData?

Comment (froesef) [reply modules/m4d-implicit-copy-stream/values.yaml:39]: Yes I thought about it and did another PR for the mover. The ChangeData option makes most sense here as a default.
With this configuration we would take Kafka data (which is always a key,value pair) and only take the value part and write this  (e.g. as JSON or parquet) to COS.

**Review by shlomitk1 — APPROVED** [review]

**Review by shlomitk1 — CHANGES_REQUESTED** [review]
Please rebase. Some of the files have been renamed as a result of the project renaming.

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**